components/SidebarOrder.tsx

```diff
@@ -0,0 +1,67 @@
+import { faHeart } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+
+import data from "../order.json";
```

Comment:
No need for this, the sorting strategy should be in-memory only, and since there are only two strategies, they can be an `enum` or another similar "bounded" type.